enforcer_interface.go

```diff
@@ -131,6 +131,7 @@ type IEnforcer interface {
 
 	UpdatePolicy(oldPolicy []string, newPolicy []string) (bool, error)
 	UpdatePolicies(oldPolicies [][]string, newPolicies [][]string) (bool, error)
+	UpdateFilteredPolicies(newPolicies [][]string, fieldIndex int, fieldValues ...string) (bool, error)
```

Comment:
You should also add the `UpdateFilteredPoliciesSelf(newPolicies [][]string, fieldIndex int, fieldValues ...string) (bool, error)` to `IDistributedEnforcer` interface.